scripts/check_fix_network.sh

```diff
@@ -146,10 +146,10 @@ function check_with_timeout() {
         restored=$RESTORED_AND_NEED_START_OVER
         break;
       else
-        skip=$([ ! -z "$reboot" ] && 'skipped')
+        [ 0 = "$reboot" ] && skip='skipped' || skip=''
```

Comment:
Although '=' works here, it is better to use '-eq' if we know for sure it is a test on non-empty integer.
```
[ 0 -eq $reboot ] && ...
```